plugins/modules/ipasudorule.py

```diff
@@ -91,6 +91,23 @@
     description: Cammand category the sudo rule applies to
```

Comment:
Also the `cmd` and `cmdgroup` options need to removed and the new options `allow_sudocmd`, `allow_sudocmdgroup`,  `deny_sudocmd` and `deny_sudocmdgroup` need to be added here.

Reply:
Fixed.